### PR TITLE
PMM-4879 Added defaults file.

### PR DESCRIPTION
**Pull request by pkadej**
[PMM-4879](https://jira.percona.com/browse/PMM-4879)

Build: [SUBMODULES-2465](https://github.com/Percona-Lab/pmm-submodules/pull/2465)

- [pmm-admin](https://github.com/percona/pmm-admin/pull/199)
- [pmm-managed](https://github.com/percona/pmm-managed/pull/1068)
- [pmm-agent](https://github.com/percona/pmm-agent/pull/356)

### CODE REVIEW

**Review by BupycHuk — APPROVED** [review]

File: api/agentpb/agent.proto [hunk 1]
```diff
@@ -391,6 +391,27 @@ message PBMSwitchPITRResponse {
...
+message ParseDefaultsFileResponse {
+
+  // Error message if parse failed.
+  string error = 1;
+
+  string username = 2;
+  string password = 3;
+  string host = 4;
+  uint32 port = 5;
+
```
Comment (BupycHuk) [inline api/agentpb/agent.proto:413]: my.cnf might contain path to socket, so we should return it too

**Review by BupycHuk — CHANGES_REQUESTED** [review]

File: api/managementpb/mysql.proto [hunk 2]
```diff
@@ -95,6 +95,8 @@ message AddMySQLRequest {
   repeated string disable_collectors = 24;
   // Custom password for exporter endpoint /metrics.
   string agent_password = 28;
+  // Path to .cnf file which contains user-name and password
+  string defaults_file = 29;
```
[the remark this answers is not in the document]
Comment (pkadej) [reply api/managementpb/mysql.proto:95]: Changed.

File: api/agentpb/agent.proto (hunk 1 above)
Comment (pkadej) [reply api/agentpb/agent.proto]: Added.

**Review by BupycHuk — DISMISSED** [review]

**Review by ritbl — APPROVED** [review]

**Comment by pkadej** [comment]
Im going to create PR from non-forked repository.

**Pull request closed without merging**